features/postmessage.yml

```
@@ -0,0 +1,16 @@
+name: PostMessage
```

I don't see this name being widely used. I think it's actually called this:

```suggestion
name: Cross-document messaging
```

(I think the file name should change to match, too.)

---

I noted this in the PR comment - I found that mostly just documentors and implementers call it `Cross-document messaging`.

In my experience, developers even *verb* `postMessage`, which I kinda love.

Happy to disagree and commit if you think we should go with the spec name, can always iterate later if we decide it's not working.

---

OK, I'm willing to go along here, but a few things need to happen:

- Link to an example of a developer verbing of `postMessage` — I would like to have an example, I think it'd make a good illustration for the docs.
- Set the case of this name to the case of the API

  ```suggestion
  name: postMessage
  ```

- _Also known as_ the "cross-document messaging" (see suggestion below).

---

https://www.startpage.com/sp/search?query=%22postmessaging%22

Sometimes it's like "postmessaging between iframes", things like that.

> Check out the MDN web documentation on [iframes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) and [postmessaging](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) to familiarize yourself with these concepts.

from https://cloud.google.com/looker/docs/embed-getting-started?hl=en